docs/notebooks/JAX_for_the_impatient.ipynb

```diff
@@ -0,0 +1,1293 @@
+{
```

Comment:
<p>Replace "you will not get any randomness" to "you'll get the same random output each time"? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Yes rereading it's indeed a weird sentence</p>